tests/repositories/test_legacy_repository.py

```diff
@@ -332,3 +333,17 @@ def test_get_4xx_and_5xx_raises(http):
     for endpoint in endpoints:
         with pytest.raises(RepositoryError):
             repo._get(endpoint)
+
+
+def test_get_redirect_has_redirect(http, monkeypatch):
+    repo = MockHttpRepository({"/foo": 200}, http)
+    redirect_url = "http://legacy.redirect.bar"
+
+    def get_mock(url):
```

Comment:
Not mandatory, but is it possibly to get this done by registering a redirected response via `httpretty`? This might be good for consistency with rest of the test cases.

Reply:
I tried to do that initially, but the redirect URL never worked for whatever reason.  I wasn't able to decipher from the `httpretty` docs or code how or if redirected responses were supported.